### ISSUE TITLE
Include loader.elf artifact in future releases

### ISSUE THREAD

**Issue by paulreimer** [cloudius-systems/osv#1118]
Including the `loader.elf` file as an artifact in future releases would help for debugging images based on the released `osv-loader.qemu`. `scripts/loader.py` could also be helpful since it is somewhat standalone and used with gdb as well.

**Comment by wkozaczuk** [comment]
The `loader.elf` is quite large (~100MB) and given we publish and upload the artifacts (like osv-loader.qemu which in reality is loader-stripped.elf) for each commit to github, we need to be mindful that is feasible. The gzipped version is around ~30MB so maybe we can start doing it.

**Comment by paulreimer** [comment]
Fair enough! I'm (quite new to the ecosystem and) trying to debug a capstan project build using the default image. IIUC I need the `loader.elf` used (I assume stripped is fine if just trying to debug my application and I'm not having any OSv problems?)

Is there a handy way to get that out of the `osv-loader.qemu` file or other artifacts? I've had some troubles trying to mount the images myself with manual qemu commands. If there's a workflow for debugging existing capstan images, that would solve my issue and the extra artifact wouldn't be necessary in that case.
